### PR TITLE
fix(typehint): use redundant alias imports for mypy

### PR DESCRIPTION
**Pull request by adam-grant-hendry**
This commit also better aligns the repo with `pyvista/CONTRIBUTING.md`.

Fixes Issue #163

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -67,3 +67,11 @@ docs/external_examples.rst
...
 # vim
 *.swp
+
+<<<<<<< Updated upstream
+# venv
+.venv
+=======
+# virtual environments
+.venv/
+>>>>>>> Stashed changes
```
Comment (tkoyama010) [inline .gitignore:77]: Conflict seems not solved.

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
Thank you for your contribution.

File: .gitignore (hunk 1 above)
Comment (adam-grant-hendry) [reply .gitignore:77]: Ya, nearly all the tests are failing for me in this repo and several simply do not run:

image

Comment (adam-grant-hendry) [reply .gitignore:77]: I think this will need some TLC ❤️

**Comment by adam-grant-hendry** [comment]
I need to do some cleanup (something went wrong when I forked the repo). I will close this and open a new PR.

**Pull request closed without merging**